src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java

```diff
@@ -6,8 +6,27 @@
 
 public class ApplicationTest {
 
+    private static final String ERROR = "" +
```

Comment:
Duplicate code. Just check that the string that gets returned contains "Invalid argument" instead.

Reply:
- I refactored the name from ERROR to INVALID_ARGUMENT_ERROR_MESSAGE.
- I extracted the text:
"Error: Invalid argument:\n" +
"$ moneymaker help\n" +
"for more info."; 
- and made it into a constant String so that I could check that this text was generated when using logger.getMessage().
- my rational for including. However...
- ...Are you suggesting I should replace this with
`logger.getMessage().contains("Invalid argument")`